src/detours.cpp

```diff
@@ -1393,6 +1393,17 @@ PVOID WINAPI DetourAllocateRegionWithinJumpBounds(_In_ LPCVOID pbTarget,
 
     *pcbAllocatedSize = DETOUR_REGION_SIZE;
     return pbNewlyAllocated;
+}
+
+BOOL WINAPI DetourIsFunctionImported(_In_ PBYTE pbCode,
+                                     _In_ PBYTE pbAddress)
+{
+    if (detour_is_imported(pbCode, pbAddress))

```

Comment:
`return detour_is_imported...`?

Reply:
I assume that the original implementor was not aware that the `bool` to `BOOL` conversion is safe per C++ standard.  I mostly just copied the code as is and I can update it as necessary.